core/server/api/canary/utils/validators/input/posts.js

```diff
@@ -28,15 +29,38 @@ const validateVisibility = async function (frame) {
     }
 };
 
+const validateLabsFeatures = async function (frame) {
+    if (!frame.data.posts || !frame.data.posts[0]) {
+        return Promise.resolve();
+    }
+
+    if (!labs.isSet('featureImageMeta')) {
```

Comment:
Wondering if we should add a similar if block in the importer :thinking:

Reply:
Couldn't that result in data loss? I don't see any harm importing the fields if the feature can be re-enabled or will be made GA in the future.